src/Microsoft.Data.Sqlite.Core/SqliteCommand.cs

```diff
@@ -246,9 +246,7 @@ public override void Prepare()
             {
                 using (var enumerator = PrepareAndEnumerateStatements(timer).GetEnumerator())
                 {
-                    while (enumerator.MoveNext())
```

Comment:
We don't use this coding pattern. Please review coding guidelines of this project first.

Reply:
Fixed it, thanks!